client/java/build.gradle

```diff
@@ -63,10 +64,13 @@ dependencies {
     implementation 'org.apache.httpcomponents:httpclient:4.5.14'
     implementation 'commons-logging:commons-logging:1.3.0'
     implementation 'org.slf4j:slf4j-api:1.7.36'
+    implementation "io.micrometer:micrometer-core:${micrometerVersion}"
```

Comment:
This has some extra dependencies. Moreover this can clash with users having micrometer already installed on their class path. Could we assume users provide this manually? Anyway they need to add jar specific to a micrometer backend they chose.